api/http/management_filetransfer.go

```diff
@@ -106,7 +108,7 @@ func (h ManagementController) getFileTransferParams(c *gin.Context) (*fileTransf
 		return nil, http.StatusConflict, app.ErrDeviceNotConnected
 	}
 
-	if c.Request.Body == nil {
+	if c.Request.Method != http.MethodGet && c.Request.Body == nil {
```

Comment:
Are we planning to keep the POST? If not, I think you can remove this if branch.

Reply:
Isn't this function also used by the upload end-point?

Reply:
Ahh, right! I already forgot about that :man_facepalming: